app/services/unleash_feature_flags.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# Service for accessing Unleash feature flags
+# Provides a centralized way to check feature flags with proper defaults
+module UnleashFeatureFlags
+  # Feature flag names
+  KESSEL_ENABLED = 'kessel_enabled'
+
+  class << self
+    # Check if a feature flag is enabled in Unleash
+    # @param flag_name [String] The name of the feature flag
+    # @param default [Boolean] Default value if Unleash is unavailable or flag doesn't exist
+    # @param context [Hash] Additional context for the flag evaluation (e.g., user_id, org_id)
+    # @return [Boolean] Whether the feature flag is enabled
+    def enabled?(flag_name, default: false, context: {})
+      return default unless unleash_available?
+
+      begin
+        # unleash-client 6.3.0 method signature: is_enabled?(flag_name, context = {}, default = false)
+        # Context can be empty hash if not needed
+        $unleash.is_enabled?(flag_name, context || {}, default)
+      rescue StandardError => e
+        Rails.logger.error("Failed to check Unleash flag '#{flag_name}': #{e.message}")
+        default
+      end
+    end
+
+    # Check if Kessel is enabled via Unleash feature flag
+    # @param context [Hash] Additional context for the flag evaluation
+    # @return [Boolean] Whether Kessel is enabled
+    def kessel_enabled?(context: {})
```

Comment:
I'd suggest to keep the service generic and not creating a method for `kessel_enabled?`, and use `enable?` instead.